### PR TITLE
feat(frontend): Add photo block

### PR DESCRIPTION
**Pull request by aarushik93**
### Background

Needed so we can have photos as inputs

### Changes 🏗️
Added a photo node that is rendered for photo analysis blocks

Only for the new autogpt platform, currently in autogpt_platform/

- Create from scratch and execute an agent with at least 3 blocks
- Import an agent from file upload, and confirm it executes correctly
- Upload agent to marketplace
- Import an agent from marketplace and confirm it executes correctly
- Edit an agent from monitor, and confirm it executes correctly

### CODE REVIEW

File: autogpt_platform/frontend/src/components/CustomNode.tsx [hunk 1]
```diff
@@ -535,6 +536,25 @@ export function CustomNode({ data, id, width, height }: NodeProps<CustomNode>) {
       ),
     );
   console.debug(`Block cost ${inputValues}|${data.blockCosts}=${blockCost}`);
+  if (data.blockType === "PhotoAnalysisBlock" ) {
```
Comment (aarushik93) [inline autogpt_platform/frontend/src/components/CustomNode.tsx]: lmk if there's a better way to filter here

**Comment by Swiftyos** [comment]
Please can you add a video demonstrating the new ui elements

File: autogpt_platform/frontend/src/components/CustomNode.tsx [hunk 2]
```diff
@@ -535,129 +536,152 @@ export function CustomNode({ data, id, width, height }: NodeProps<CustomNode>) {
...
+            <SchemaTooltip description={data.description} />
+          </div>
+        </div>
+        <PhotoNodeComponent
+          data={data}
+          handleInputChange={handleInputChange}
+          generateOutputHandles={generateOutputHandles}
+        />
+      </div>
+    );
```
Comment (majdyz) [inline autogpt_platform/frontend/src/components/CustomNode.tsx:560]: Should have stopped the if block here.

**Comment by aarushik93** [comment]
Gonna clean this up and do it from my original branch

**Pull request closed without merging**